app/components/SelectWallet.tsx

```diff
@@ -24,7 +30,7 @@ export default function SelectWallet() {
           <span>
             {connector.name === "Injected" ? "Browser Wallet" : connector.name}
             {!connector.ready && " (unsupported)"}
-            {isConnecting &&
+            {isLoading &&
```

Comment:
same but different lol

Reply:
Just a few slight renamings and data being swapped between hooks by wagmi here